frontend/tests/jest/__mocks__/products.ts

```diff
@@ -3174,3 +3174,124 @@ export const mockedLayoutProps: Pick<ProductTemplateProps, 'layoutProps'> = {
       ],
    },
 };
+
+export const mockedReviews: ProductReview[] = [
+   {
+      reviewid: 37443,
+      rating: 5,
+      headline: null,
+      productName: 'Precision Cleaning Kit',
+      productVariantName: 'New',
+      body: 'As advertised and solved our problem.',
+      date: '<time   title="Thu, 03 Nov 2022 04:19:46 -0700" datetime="2022-11-03T04:19:46-07:00">Nov 3, 2022</time>',
+      created_date: 1667474386,
+      modified_date: 1667474465,
+      langid: 'en',
+      author: {
+         userid: 3081773,
+         name: 'MARVIN STABLER',
+         avatar:
+            'https://www.cominor.com/static/images/avatars/User/ifixit/avatar-9.thumbnail',
+         url: 'https://www.cominor.com/User/3081773/MARVIN+STABLER',
+         canEdit: true,
+      },
+   },
+   {
+      reviewid: 37196,
+      rating: 5,
+      headline: null,
```

Comment:
Looks like all the headlines are null. We may want to include at least one review with a headline.

Reply:
Got it! I'll add a headline to one of them.

Reply:
Updated in https://github.com/iFixit/react-commerce/pull/1173/commits/ef3794b238643703f10ff9dc5a40f9bb6037dce4.